code/datums/uplink_item.dm

```diff
@@ -143,6 +143,16 @@ var/list/uplink_items = list()
 	cost = 10
 	jobs = list("Chemist", "Medical Doctor", "Chief Medical Officer", "Geneticist", "Virologist")
 
+/datum/uplink_item/job_specific/HumanPokeball
+	name = "Rapid Human Restraint Device"
+	desc = "This device, when thrown will rapidly apply a blindfold, muzzle and straight jacket to a target, specialised bluespace devices and clever use of springs will remove any clothing they have which might impede this process. The force is sufficient enough to knock the target down for a couple of seconds, be sure to grab hold of them quickly as while they will be unable to unbind themselves without help there is nothing stopping your target from running away."
```

Comment:
Very awful run on sentence here. Reword it please.
Remember, if you use commas as parenthesis them sentence should still read alright after removing the parenthesis info.